### ISSUE TITLE
**Find the number of tweets per location and per date**

### ISSUE THREAD

**Issue by edogdu** [edogdu/covid19#2]
Count the COVID tweets by location and date.

Code: **find_tweets_by_date_location.ipynb**

Output: tweet_count_loc_date.csv
location, date, count

Due: Aug 31
Who: @thuynh3-minh

**Comment by thuynh3-minh** [comment]
Hello!
I could not upload a file here so I will post a link [https://github.com/thuynh3-minh/testUPload/blob/master/tweets_per_day.ipynb](url)
it counts the number of tweets per day. Feel free to make any comments. I can show the location for one file but not many files so I need your help on that.

**Comment by edogdu** [comment]
Minh,

Your program needs to be called: https://github.com/thuynh3-minh/testUPload/blob/master/tweets_per_day.ipynb

Put comments in your program, explain steps.

There is no results in the program, it should report **#tweets by day and location**.

Why can't you upload the program to project code area?

**Comment by thuynh3-minh** [comment]
Hi dr Dogdu,
What time are you available today? Can I come or start a colaborate to ask
for your help?

As for tweets by day and location, what exactly should the output look like?

I created a sample file based on the tweets on a few files and it shows a
dictionary like this {day: numoftweets}. I dont know if it is good or not.

Thank you

**Comment by thuynh3-minh** [comment]
A quick update on the code. I was able to update the time and output to a csv file. .Right now it looks like the image I attached, unsorted and not yet counted. Please me know if there is any changes that should be made. Thank you!

I will do the following
1. extract the names of states and country (tx=> texas; us=> United States) so it matches with the other dataset.
2. Sort and count by date and citycode.

![5](https://user-images.githubusercontent.com/65977877/94893569-21bd9e80-044d-11eb-8f10-a26fbb0a89b0.PNG)

**Comment by thuynh3-minh** [comment]
One more thing, I should change the time to form DD/MM/YYYY.
(Btw, the first row to the left is the cityID)

**Comment by edogdu** [comment]
This looks good. Please check the data here: 

[JHU daily data](https://github.com/CSSEGISandData/COVID-19/tree/master/csse_covid_19_data/csse_covid_19_daily_reports)

We will compare against this data. So, your data should be in the same format.

**Comment by thuynh3-minh** [comment]
That sounds good. Thank you!

**Comment by edogdu** [comment]
Please fill out your screening before 2pm :) Hope you are okay!

Take care.

--
Erdoğan Doğdu, Ph.D.

**Comment by thuynh3-minh** [comment]
Yes, sir. I am so sorry. I am always good I just barely woke up at that
time. I will remember to do it.
Have a good weekend.
